### ISSUE TITLE
Block performance test

### ISSUE THREAD

**Issue by senier** [Componolit/ada-components#3]
Implement a block client that measures the good-case throughput of a block server. A test works as follows:

- For request type (in read/write):
   - For request size (between device block size .. 2 MB (?)):
      - For number of requests:
          - Send request of request type
      - Wait for all request to finish
      - Calculate throughput

- For all events handling a response or request:
   - Record queue size (requests in flight) and time stamp

Requests are performed linearly in ascending order.

**Comment by jklmnn** [comment]
The throughput test should be configurable with:
 - request size
 - maximum data
 - operation
 - buffer size

The results are provided in XML and log output. The log output is used to check the liveliness and contains preprocessed data (req/s, mb/s, mb total, etc). The XML data contains the current timestamp and the requests received until this date:
```xml
<test request_size="n" total_size="n" operation="read|write" buffer_size="n" cache_size="n">
   <sample time="n" reqs="n"/>
</test>
```

**Comment by jklmnn** [comment]
Configuration format:
```xml
<test request_size="n" operation="read|write" data_size="n" buffer_size="n" device="dev"/>
```

**Comment by jklmnn** [comment]
Fixed by #5.
